zubhub_frontend/zubhub/public/locales/en/translation.json

```diff
@@ -1190,5 +1190,11 @@
       "text": "Get inspired by thousands of creators around the world.",
       "buttonLabel": "Get Started"
     }
+  },
+  "sessionExpired": {
```

Comment:
For this, as you know, every PR that was merged into `master`, I had to rebase. Then at some point when fixing conflicts, I lost some of the changes and they where breaking stuff. I added them manually

Reply:
The sessionExpired PR was merged and later reverted due to blocking regression which I had fixed in this branch, so instead of having Steve do it again, and yet still I had to change it again since he was working from the old dependencies, I decided to keep the changes when rebasing

Reply:
the sessionExpired PR was discarded because it wasn't behaving as expected. We can't have it here since that PR has already been reverted. Unless someone reworked it and submitted a new PR (in that case we can merge it and rebase this on that), we still can't have it here because it'd be introducing a new feature that is not part of this PR